web_src/js/components/DiffFileTreeItem.vue

```diff
@@ -122,6 +127,8 @@ span.svg-icon.octicon-diff-renamed {
 div.directory {
   display: grid;
   grid-template-columns: 18px 20px auto;
+  user-select: none;
```

Comment:
Why `user-select: none;`? Why shouldn't users be able to select the text of a directory?

Reply:
I mean, I can see two ways forward: Either, everything is selectable, or nothing in that tree is selectable, so apply this for files too.
From an accessibility point of view, it's probably best anyways longterm to convert the `div`s/`a`s into a `button`. Then, neither of these two selection changes will be needed.
But that's for later.
For now, it should suffice to give both `files` and `directories` the `unselectable` class.

Reply:
Tried to make it a button, but got frustrated with the padding. 

And as it stands you can't select the files, I think because they are links. 

So only directories require user-select: none, if I don't do that double clicking them would select stuff all the time which makes using them just annoying.